.gitbook/overview/roadmap.md

```diff
@@ -46,19 +46,19 @@
   - DAO
   - Cross-chain NFT (compatible with NFT marketplaces)
 
-### 2023 Q3: Healthcare Services on Data Market Protocol
+### 2023 Q3: Healthcare Services on Data Exchange Protocol
 
-- Release data market protocol v1 on Panacea mainnet
-- Integrate MediBloc healthcare services with data market
+- Release data exchange protocol v1 on Panacea mainnet
+- Integrate MediBloc healthcare services with data exchange protocol
 - Revise whitepaper
```

Comment:
```suggestion
- Update and release new version of whitepaper
```

Reply:
Thanks Youngjoon! The change looks great and I just added a slight suggestion to change our wording regarding the whitepaper revision.

Reply:
done https://github.com/medibloc/panacea-core/commit/a1d623682500a823ff80808dbb4b0f9e89786bf0